resources/user-guides/getting-started-with-adobe-experience-platform/README.md

```diff
@@ -39,16 +39,16 @@ To start collecting data in Adobe Experience Platform, you need to create an XDM
    * Select `Next`
    * Set a name for this dataset and select `Finish`.
 
-### Generate an Experience Edge environment identifier <a id="generate-an-experience-edge-environment-identifier"></a>
+### Generate a Datastream identifier <a id="generate-a-datastream-identifier"></a>
```

Comment:
Can you verify if other pages are referencing this anchor, and if so update those links?

Reply:
yes, didn't find any